src/AWSS3.jl

```diff
@@ -598,7 +604,12 @@ function s3_put_tags(aws::AbstractAWSConfig, bucket, tags::SSDict; kwargs...)
     return s3_put_tags(aws, bucket, "", tags; kwargs...)
 end
 
-s3_put_tags(a...) = s3_put_tags(global_aws_config(), a...)
+function s3_put_tags(bucket, path, tags; kwargs...)
+    return s3_put_tags(global_aws_config(), bucket, path, tags; kwargs...)
+end
+function s3_put_tags(bucket, tags; kwargs...)
+    return s3_put_tags(global_aws_config(), bucket, tags; kwargs...)
+end
```

Comment:
Makes more sense to use `args...` here to avoid having to specify additional methods:
```suggestion
s3_put_tags(args...; kwargs...) = s3_put_tags(global_aws_config(), args...; kwargs...)
```